df_py/util/dftool_module.py

```diff
@@ -389,6 +385,12 @@ def do_predictoor_data():
     st_block, fin_block = get_st_fin_blocks(chain, arguments.ST, arguments.FIN)
 
     # main work
+    predictoor_contracts = query_predictoor_contracts(chain_id)
+
+    # retry_function(
+    #     query_predictoor_contracts, arguments.RETRIES, 10, chain_id
+    # )
```

Comment:
Not sure whether to comment on this or not. I'm sure this is for debugging purposes... maybe wrap it in an arg?

i.e. `do_predictoor_data(retry = false)`